src/main/java/org/jabref/gui/util/ThemeLoader.java

```diff
@@ -37,12 +38,19 @@
 public class ThemeLoader {
 
     private static final String DEFAULT_PATH_MAIN_CSS = JabRefFrame.class.getResource("Base.css").toExternalForm();
-    private static final String CSS_SYSTEM_PROPERTY = System.getProperty("jabref.theme.css");
     private static final Logger LOGGER = LoggerFactory.getLogger(ThemeLoader.class);
+    private final String CSS_SYSTEM_PROPERTY;
     private final FileUpdateMonitor fileUpdateMonitor;
 
     public ThemeLoader(FileUpdateMonitor fileUpdateMonitor) {
         this.fileUpdateMonitor = Objects.requireNonNull(fileUpdateMonitor);
```

Comment:
Please pass the JabRef Preferences object as parameter, we try to avoid direct access of dependencies. Instead we try to adhere to dependency injection